app/assets/javascripts/darkswarm/services/products.js.coffee

```diff
@@ -24,6 +24,10 @@ Darkswarm.factory 'Products', ($resource, Enterprises, Dereferencer, Taxons, Pro
         if product.variants?.length > 0
           prices = (v.price for v in product.variants)
           product.price = Math.min.apply(null, prices)
+          if product.variants.length > 1
```

Comment:
I'm not good with js but maybe this can be factored out into a separate method to reduce the if-nesting levels.